api_tests/dry/rfc003/test.js

```diff
@@ -206,12 +206,17 @@ describe("RFC003 HTTP API", () => {
                     swap.protocol.should.equal("rfc003");
                     swap.state.should.equal("Start");
                     let links = swap._links;
-                    links.self.href.should.be.oneOf([swap_url_1, swap_url_2]);
+                    links.self.href.should.be.oneOf([
```

Comment:
❓Is it really just `onOf` or `allOf`?

Reply:
We are checking that each swap returned by GET /swaps (in this case 2) matches with one of the swaps sent by Alice. Because we store the swaps using a `HashMap` they're not kept in chronological order, so this is a way to check it.